starknet_py/net/full_node_client.py

```diff
@@ -114,7 +114,7 @@ async def get_block_traces(
     async def get_events(
         self,
         address: Hash,
-        keys: List[List[str]],
+        keys: Optional[List[List[str]]] = None,
```

Comment:
Let's allow for this to be a list of integers, and convert it to felts ourselves